compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt

```diff
@@ -70,6 +70,12 @@ internal class SkiaParagraph(
         }
 
     init {
+        // Size is not known until layout is complete but to apply it, we need to re-create
+        // skia's paragraph :'(
+        // layouter might use cached instance if no [ShaderBrush] was applied.
+        layouter.setBrushSize(Size(width, height))
```

Comment:
Width and height can be changed later. We need to set it after every layout inside paint:

```
layoutParagraph(width = width)
layouter.setBrushSize(Size(width, height))
layoutParagraph(width = width)
```

Reply:
It's already handled here:

https://github.com/JetBrains/compose-multiplatform-core/blob/ba23164162d2c036c9c34b645a82bc4d1d226747/compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/SkiaParagraph.skiko.kt#L534